android/src/main/java/com/swmansion/reanimated/nodes/JSCallNode.java

```diff
@@ -19,12 +19,17 @@ public JSCallNode(int nodeID, ReadableMap config, NodesManager nodesManager) {
   @Override
   protected Double evaluate() {
     WritableArray args = Arguments.createArray();
-    for (int i = 0; i < mInputIDs.length; i++) {
-      Node node = mNodesManager.findNodeById(mInputIDs[i], Node.class);
+    for (int mInputID : mInputIDs) {
```

Comment:
minor tweak.

Reply:
AFAIK this actually allocates an iterator, so it is not merely a tweak